Source/JavaScriptCore/b3/B3LowerToAir.cpp

```diff
@@ -2325,7 +2325,7 @@ class LowerToAir {
         }
         
         auto tryShl = [&] (Value* shl, Value* other) -> bool {
-            std::optional<unsigned> scale = scaleForShl(shl, offset);
+            std::optional<unsigned> scale = scaleForShl(leaOpcode, shl, offset);
```

Comment:
Is `leaOpcode` correct?

Reply:
Again, I think so? Not sure what the other candidates would be?